packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -838,3 +950,113 @@ The Authenticator object defines the following properties:
   }
 }
 ```
+
+#### Example Duo Authenticator
+
+```
+{
+    "type": "app",
+    "id": "aut5gnvcjUHIWb25J0g4",
+    "key": "duo",
+    "status": "ACTIVE",
+    "name": "Duo Security",
+    "created": "2022-07-15T21:14:02.000Z",
+    "lastUpdated": "2022-07-15T21:14:02.000Z",
+    "settings": {},
```

Comment:
Not sure why there is an empty settings object - normally should be omitted ...

Reply:
Will check. This was returned in the response, so included it here.